src/nupic/regions/CLAClassifierRegion.py

```diff
@@ -198,7 +207,12 @@ def __init__(self,
                maxCategoryCount=None
                ):
 
+    # Set default implementation
+    if implementation is None:
+      implementation = Configuration.get('nupic.opf.claClassifier.implementation')
+
     # Convert the steps designation to a list
+    self.classifierImp = implementation
```

Comment:
SPRegion uses `spatialImp` and TPRegion uses `temporalImp`, so I chose `classifierImp` to be consistent with that. Would you still change?